projects/plugins/jetpack/_inc/lib/class-jetpack-ai-helper.php

```diff
@@ -95,13 +102,35 @@ public static function transient_name_for_completion() {
 		return 'jetpack_openai_completion_' . get_current_user_id(); // Cache for each user, so that other users dont get weird cached version from somebody else.
 	}
 
+	/**
+	 * Mark the edited post as "touched" by AI stuff.
+	 *
+	 * @param  int $post_id Post ID for which the content is being generated.
+	 * @return void
+	 */
+	private static function mark_post_as_ai_assisted( $post_id ) {
+		if ( ! $post_id ) {
+			return;
+		}
+		$previous = get_post_meta( $post_id, self::$post_meta_with_ai_generation_number, true );
+		if ( ! $previous ) {
+			$previous = 0;
+		} elseif ( ! is_numeric( $previous ) ) {
+			// Data corrupted, nothing to do.
+			return;
+		}
+		$new_value = intval( $previous ) + 1;
+		update_post_meta( $post_id, self::$post_meta_with_ai_generation_number, $new_value );
```

Comment:
Here we count the number of times a post used the AI blocks to generate content? I'm curious of this would reflect what we are aiming for since an author could generate AI continent multiple times for a post but never use the content.

Either way, I think this is a great start to what we want to achieve

Reply:
> I'm curious of this would reflect what we are aiming for since an author could generate AI continent multiple times for a post but never use the content.

Yes, correct, that could happen